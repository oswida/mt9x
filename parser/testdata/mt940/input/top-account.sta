:20:0806120035300006
:25:987654321
:28C:112/1
:60F:C080611SEK89324,89
:61:0806120611C1606,NTRF111222333//6091 000001
:61:0806120611C57392,84NMSC444555666//6000 IT-A06
:61:0806120611C74697,NTRF111222333//6091 GI
:61:0806110611C4016,65NTRF777888999//6000 IT-A06
:61:0806110611D4016,65NTRF444555666//6000 IT-A06
:61:0806110611C874095,NTRF111222333//6000 IT-A06
:61:0806110611D874095,NTRF888999777//6000 IT-A06
:61:0806120611C426709,NTRF111222333//6091 BGINB
:61:0806100611D14412,NMSC777888999//6000 FIL-E
:61:0806100611D7500,05NMSC444555666//6000 FIL-E
:61:0806100611D1058317,5NMSC111222333//6000 FIL-E
:61:0806100611D214464,NMSC888999777//6000 FIL-E
:61:0806100611D2114,NMSC555666777//6000 FIL-E
:61:0806110611C3099048,NTRF555666777//6000 IT-A06
:61:0806110611D3099048,NTRF555666777//6000 IT-A06
:61:0806110611D125,NCHG111222333//0000 AVGIFT
:61:0806120611C72941,NTRF444555666//60001ABOL
:61:0806110611C53422,NTRF444555666//60001ABOL
:61:0806120611C74764,NTRF444555666//60001ABOL
:61:0806120611C183165,NTRF444555666//60001ABOL
:61:0806120611C17066,NTRF777888999//60001ABOL
:61:0806120611C49735,7NTRF777888999//60001ABOL
:62F:D080611SEK196109,12
:64:D080611SEK1154185,66