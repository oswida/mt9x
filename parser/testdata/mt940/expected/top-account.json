{
 "tag20": "0806120035300006",
 "tag25": {
  "account": "987654321"
 },
 "tag28": {
  "stmt_number": "112",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2008-06-11T00:00:00Z",
  "currency": "SEK",
  "amount": "89324.89"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "1606",
    "trx_ident": "NTRF",
    "owner_ref": "111222333",
    "institution_ref": "6091 000001"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "57392.84",
    "trx_ident": "NMSC",
    "owner_ref": "444555666",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "74697",
    "trx_ident": "NTRF",
    "owner_ref": "111222333",
    "institution_ref": "6091 GI"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "4016.65",
    "trx_ident": "NTRF",
    "owner_ref": "777888999",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "4016.65",
    "trx_ident": "NTRF",
    "owner_ref": "444555666",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "874095",
    "trx_ident": "NTRF",
    "owner_ref": "111222333",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "874095",
    "trx_ident": "NTRF",
    "owner_ref": "888999777",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "426709",
    "trx_ident": "NTRF",
    "owner_ref": "111222333",
    "institution_ref": "6091 BGINB"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-10T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "14412",
    "trx_ident": "NMSC",
    "owner_ref": "777888999",
    "institution_ref": "6000 FIL-E"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-10T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "7500.05",
    "trx_ident": "NMSC",
    "owner_ref": "444555666",
    "institution_ref": "6000 FIL-E"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-10T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "1058317.5",
    "trx_ident": "NMSC",
    "owner_ref": "111222333",
    "institution_ref": "6000 FIL-E"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-10T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "214464",
    "trx_ident": "NMSC",
    "owner_ref": "888999777",
    "institution_ref": "6000 FIL-E"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-10T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "2114",
    "trx_ident": "NMSC",
    "owner_ref": "555666777",
    "institution_ref": "6000 FIL-E"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "3099048",
    "trx_ident": "NTRF",
    "owner_ref": "555666777",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "3099048",
    "trx_ident": "NTRF",
    "owner_ref": "555666777",
    "institution_ref": "6000 IT-A06"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "D",
    "amount": "125",
    "trx_ident": "NCHG",
    "owner_ref": "111222333",
    "institution_ref": "0000 AVGIFT"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "72941",
    "trx_ident": "NTRF",
    "owner_ref": "444555666",
    "institution_ref": "60001ABOL"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-11T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "53422",
    "trx_ident": "NTRF",
    "owner_ref": "444555666",
    "institution_ref": "60001ABOL"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "74764",
    "trx_ident": "NTRF",
    "owner_ref": "444555666",
    "institution_ref": "60001ABOL"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "183165",
    "trx_ident": "NTRF",
    "owner_ref": "444555666",
    "institution_ref": "60001ABOL"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "17066",
    "trx_ident": "NTRF",
    "owner_ref": "777888999",
    "institution_ref": "60001ABOL"
   }
  },
  {
   "tag61": {
    "value_date": "2008-06-12T00:00:00Z",
    "entry_date": "0000-06-11T00:00:00Z",
    "dc_mark": "C",
    "amount": "49735.7",
    "trx_ident": "NTRF",
    "owner_ref": "777888999",
    "institution_ref": "60001ABOL"
   }
  }
 ],
 "tag62": {
  "dc_mark": "D",
  "date": "2008-06-11T00:00:00Z",
  "currency": "SEK",
  "amount": "196109.12"
 },
 "tag64": {
  "dc_mark": "D",
  "date": "2008-06-11T00:00:00Z",
  "currency": "SEK",
  "amount": "1154185.66"
 }
}